### PR TITLE
basic setup for use survey hook

### PR DESCRIPTION
**Pull request by vaibhavg15**

### CODE REVIEW

File: src/hooks/useSurvey.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+import { API_URLS } from '../service/apiUrls';
+
+// TODO Remove this and import from uikit
+enum LOADING_STATUS {
+  NOT_YET_STARTED= 0,
+  LOADING= 1,
+  COMPLETED= 2,
+  FAILED= 3,
+}
+
```
Comment (shanu798) [inline src/hooks/useSurvey.tsx:18]: update this from uikit

File: src/hooks/useSurvey.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  );
+
+  const onSubmit = useCallback(
+    () => new Promise((
+      resolve, reject,
+    ) => {
+      // if (isLoading(submitStatus)) {
+      //   return;
+      // }
+      setSubmitStatus(LOADING_STATUS.LOADING);
```
Comment (shanu798) [inline src/hooks/useSurvey.tsx:60]: update the loading status also

**Review by shanu798 — COMMENTED** [review]
review done

**Comment by rohan-1898** [comment]
review done

**Pull request merged**